test/integration/targets/noexec/runme.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -eux
+
+trap 'umount "${OUTPUT_DIR}/ramdisk"' EXIT
+
+mkdir "${OUTPUT_DIR}/ramdisk"
+mount -t tmpfs -o size=32m,noexec,rw tmpfs "${OUTPUT_DIR}/ramdisk"
+ANSIBLE_REMOTE_TMP="${OUTPUT_DIR}/ramdisk" ansible-playbook -i ../../inventory "$@" test-noexec.yml
```

Comment:
Does this test need to involve an actual remote host? If not, use `localhost` instead of `all` and do not use the `../../inventory` inventory file. You can provide a [non-empty](https://github.com/ansible/ansible/blob/devel/test/integration/targets/ansible-doc/inventory) inventory file to avoid inventory warnings.

If it does need to involve an actual remote, use `setup_remote_tmp_dir` to create the temporary directory, and tasks to setup/teardown the ramdisk on the remote.

Either one of those changes should allow the test to work once split controller/remote is implemented.

Reply:
Updated to use localhost